adds a tools to find gene copy number from newick file

anilthanki opened the pull request.

Commit:
adds a tools to find gene copy number from newick file

nsoranzo commented on tools/ete/ete_gene_cnv.py:
```diff
@@ -0,0 +1,87 @@
+from __future__ import print_function
+
+import optparse
+import collections
```
Import statements are in the wrong order. 'import collections' should be before 'import optparse'

nsoranzo replied:
Actually it would be good to use [argparse](https://docs.python.org/3/library/argparse.html), `optparse` has been deprecated for a while.

Commit:
fixes for GitHub comments

nsoranzo committed:
Lint for flake8

nsoranzo committed:
Disable URL linting

Keeps failing with:

```
ERROR: Error '<urlopen error ftp error: [Errno 110] Connection timed out>' accessing ftp://ftp.ncbi.nih.gov/pub/taxonomy/taxdump.tar.gz
```

Commit:
fixes test bug

nsoranzo commented on tools/ete/ete_gene_cnv.py:
```diff
@@ -0,0 +1,86 @@
+        leaves = genetree.get_leaf_names()
+
+        # Genetree nodes are required to be in gene_species format
+        leaves_list = [_ for _ in leaves if '_' in _]
```
Should we raise an issue if a leaf node doesn't contain `_` or just filter it out (as done here)?

nsoranzo committed:
Use tabular format for ete_gene_cnv output

Cleanup Python script.

nsoranzo commented:
@anilthanki I've changed the output format to tabular and revised the Python script for clarity, can you please check the changes?
